build.sbt

```diff
@@ -11,7 +11,7 @@ ThisBuild / developers := List(
 )
 ThisBuild / startYear := Option(2021)
 ThisBuild / tlSonatypeUseLegacyHost := true
-ThisBuild / tlBaseVersion := "0.2"
+ThisBuild / tlBaseVersion := "0.3"
```

Comment:
Unfortunately, due to the weird way Scala 2.12 encodes methods on `AnyVal`s, this is a binary-incompatible change. I think there's probably a convoluted way to encode it to maintain bincompat, but this library is still very early in its versioning, so I don't think it's worth it.